### PR TITLE
Feature/storage data

### PR DESCRIPTION
**Pull request by GoodWien**

### CODE REVIEW

File: libraries/chain/content_card_evaluator.cpp [hunk 1]
```diff
@@ -32,6 +33,8 @@ namespace graphene { namespace chain {
 void_result content_card_create_evaluator::do_evaluate( const content_card_create_operation& op )
 { try {
    database& d = db();
+   if (HARDFORK_REVPOP_15_PASSED(fc::time_point_sec(fc::time_point::now())))
```
Comment (tedthecreator) [inline libraries/chain/content_card_evaluator.cpp:36]: `if (HARDFORK_REVPOP_15_PASSED(head_block_time()))`

File: libraries/chain/content_card_evaluator.cpp [hunk 2]
```diff
@@ -64,6 +67,8 @@ object_id_type content_card_create_evaluator::do_apply( const content_card_creat
 void_result content_card_update_evaluator::do_evaluate( const content_card_update_operation& op )
 { try {
    database& d = db();
+   if (HARDFORK_REVPOP_15_PASSED(fc::time_point_sec(fc::time_point::now())))
```
Comment (tedthecreator) [inline libraries/chain/content_card_evaluator.cpp:70]: if (HARDFORK_REVPOP_15_PASSED(head_block_time()))

File: libraries/chain/personal_data_evaluator.cpp [hunk 3]
```diff
@@ -30,6 +31,8 @@ namespace graphene { namespace chain {
 void_result personal_data_create_evaluator::do_evaluate( const personal_data_create_operation& op )
 { try {
    database& d = db();
+   if (HARDFORK_REVPOP_15_PASSED(fc::time_point_sec(fc::time_point::now())))
```
Comment (tedthecreator) [inline libraries/chain/personal_data_evaluator.cpp:34]: `if (HARDFORK_REVPOP_15_PASSED(head_block_time()))`

**Review by tedthecreator — CHANGES_REQUESTED** [review]
Take a look at the `/libraries/protocol/custom_authorities/sliced_lists.hxx`

File: libraries/chain/content_card_evaluator.cpp (hunk 1 above)
Comment (GoodWien) [reply libraries/chain/content_card_evaluator.cpp]: Done.

File: libraries/chain/content_card_evaluator.cpp (hunk 2 above)
Comment (GoodWien) [reply libraries/chain/content_card_evaluator.cpp]: Done.

File: libraries/chain/personal_data_evaluator.cpp (hunk 3 above)
Comment (GoodWien) [reply libraries/chain/personal_data_evaluator.cpp]: Done.

**Review by tedthecreator — APPROVED** [review]

**Pull request merged**